index.html

```diff
@@ -6,11 +6,47 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+<header>
+    <img class="logo" src="images/logo.jpg" alt="logo">
+    <nav>
+        <label for="hamburger">&#9776;</label>
+        <input type="checkbox" class="hamburger"/> 
```

Comment:
If you change hamburger to be an id instead of a class, the checkbox trick will work properly. By the way, JS is usually more convenient for this.